src/ert/scheduler/job.py

```diff
@@ -349,3 +353,44 @@ def log_info_from_exit_file(exit_file_path: Path) -> None:
             *filecontents
         )
     )
+
+
+async def log_warnings_from_forward_model(real: Realization) -> None:
+    """Parse all stdout and stderr files from running the forward model
+    for anything that looks like a Warning, and log it.
+
+    This is not a critical task to perform, but it is critical not to crash
+    during this process, thus any Exception is suppressed."""
+
+    def line_contains_warning(line: str) -> bool:
+        return (
+            "Warning:" in line
+            or "FutureWarning" in line
+            or "DeprecationWarning" in line
+            or "WARNING" in line
```

Comment:
```suggestion
            "Warning" in line
            or "warning" in line
```
Would just checking this be sufficient, or would we pick up unwanted things?

Reply:
That is an open question for now :) . Probably no and yes.

Reply:
I will add some test code that makes it more specific what we want to include and not, based on what I can find in real life examples on scratch.

Reply:
Potentially we just want FutureWarnings first. Then we can add more later. The most permissive would be `re.search('warning', line, re.IGNORECASE)`.

Reply:
In one example runpath..
```
$ grep -i warning *stdout* *stderr* | wc -l
15587
```
(of which 7000 are unique).  I think we must be more specific.